src/Tribe/Aggregator/API/Origins.php

```diff
@@ -68,7 +68,7 @@ public function __construct() {
 			),
 			'url' => (object) array(
 				'id' => 'url',
-				'name' => __( 'Other URL (beta)', 'the-events-calendar' ),
+				'name' => __( 'Other URL', 'the-events-calendar' ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'name'" and double arrow, but found 1.

<sub>hash: 10317659dbde269773144a100c2da5bd</sub>